### PR TITLE
fix dynamic insert-css usage

### LINKED ISSUE #109: breaks when insert-css usage isn't naively statically analyzable
hey, so this has come up for me a few times, most recently when i used [`ngraph.pixel/lib/tooltip.js#L12`](https://github.com/anvaka/ngraph.pixel/blob/master/lib/tooltip.js#L12).

basically, if any code uses `insert-css` in a way that isn't naively statically analyzable (if it uses a variable defined elsewhere, even if it's not dynamic), `bankai` removes the `insertCss = require('insert-css')` but leaves the expression using `insertCss` that it couldn't analyze.

leaving this issue here because it happens when i use `bankai`, even though i'm guessing it's [`css-extract`](https://github.com/stackcss/css-extract)'s fault.

### PR DESCRIPTION
**Pull request by ahdinosaur**
uses forked version of `static-module`: https://github.com/substack/static-module/pull/22

fixes https://github.com/yoshuawuyts/bankai/issues/109

### CODE REVIEW

File: package.json
Comment (yoshuawuyts) [inline package.json]: Let's not merge this patch until this lands in `static-module` as github urls don't resolve from cache

File: test/index.js
Comment (yoshuawuyts) [inline test/index.js]: extra `console.log`

**Review by yoshuawuyts — DISMISSED** [review]

**Comment by jondashkyle** [comment]
Just checking in on this one. Seems like `static-module` is going to be slow to merge any pull requests 😕

**Comment by yoshuawuyts** [comment]
@jondashkyle I kinda feel that if this patch is needed too complex things are being done. Also no movement on the other patch, so think this is not going to land anytime soon (unless more work is done, but it's been a bit stale lately unforch)

**Review by goto-bus-stop — APPROVED** [review]
fixed on main branch, gonna get this test in. thanks!

**Pull request merged**
